cmd/executor/cmd/root.go

```diff
@@ -73,6 +74,16 @@ var RootCmd = &cobra.Command{
 			if len(opts.Destinations) == 0 && opts.ImageNameDigestFile != "" {
 				return errors.New("You must provide --destination if setting ImageNameDigestFile")
 			}
+
+			if additionalWhitelist == nil {
+				additionalWhitelist = []string{
```

Comment:
i think we should default this to empty string. 
if users want to remove `/var/run` then they would have to run kaniko as
`/kaniko/executor --additional-whitelist=""`  instead users who need it to whitelisted should explicitly specify it.

e.g. #903

Reply:
>if users want to remove /var/run then they would have to run kaniko as
/kaniko/executor --additional-whitelist="" instead users who need it to whitelisted should explicitly specify it.

I'm not strongly opposed to that but it seems like it would be a pretty major and invisible change for users. I can imagine users who don't realize they need to explicit add `/var/run` having their build times dramatically increase.

I think defaulting it to the current behavior would be a better user experience and let people opt in. I can see how an empty string might be a better experience for new users though.

WDYT

Reply:
I also thought about being backward compatible for existing users. 
 But we want these users to eventually correct their pod config and explicitly add `/var/run` to `additional-whitelist`. 
I would want us to push them towards that direction. 
This is challenging for us especially since users don't have to change their image tag if they are pointing to `latest`

Maybe for a couple of releases we could add a visible Warning that `/var/run` needs to be explicitly whitelisted especially if their build times have increased. 
Also monitor slack closely.

WDYT?

Reply:
linking for history https://github.com/GoogleContainerTools/kaniko/pull/975#issuecomment-577847191

Reply:
done :+1: